### ISSUE TITLE
Support vite

### ISSUE THREAD

**Issue by swarakaka** [petaki/inertia-go#6]
Hello @petaki ,

can you please match this package with vite?
[like this](https://github.com/gs205642/fiber-inertia)

with regards,
Swara

**Comment by petaki** [comment]
@swarakaka Yes, but I am working on a different solution. I don't want to introduce frontend dependencies to this package.

**Comment by swarakaka** [comment]
Hi @petaki,
thank you very much for your reply. The important thing is to have support for vite.

**Comment by noor-tg** [comment]
Any news about vite support ?

**Comment by petaki** [comment]
I migrated one sample app from Mix to Vite and will migrate the others soon. I also created and linked a small helper lib for this.

- https://github.com/petaki/satellite
- https://github.com/petaki/support-go/tree/master/vite
